.github/workflows/test.yml

```
@@ -54,3 +63,38 @@ jobs:
+
+      - name: Generate coverage
+        run: npm run coverage
+        env:
+          KANBN_ENV: test
+
+      - name: Check coverage thresholds
+        run: npm run coverage:check
+        continue-on-error: true
+
+      - name: Upload coverage artifacts
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)